.env.example

```diff
@@ -58,3 +58,8 @@ IMAGE_TAG=1.1.0
 KEYCLOAK_USER=admin
 KEYCLOAK_PASSWORD=admin
 KEYCLOAK_HOST=<<INSERT_YOUR_IP>>
+
+#---- Scenario Explorer ----
+CUT_ADDONS_DIRECTORY=../scenario-explorer-addon
+CUT_PORTAL_DIRECTORY=../scenario-explorer-portalconfig
+CUT_DOCKER_DIRECTORY=../masterportal-scenario-explorer
```

Comment:
Would all three Repos be initialized in the UMP repo as Git submodules? If so, I'd suggest to mount them in another folder to keep the UMP repo more clean and less cluttered. Scenario explorer can be one frontend, but is not necessarily **the** frontend of UMP

Reply:
No these repositories do live outside of the UMP repo. This is just relvant for development.
My local directory looks like this:

citysciencelab
├── masterportal-scenario-explorer
├── scenario-explorer-addon
├── scenario-explorer-portalconfig
└── urban-model-platform

Every directory is just a git clone of the corresponding repo.

Maybe it makes sense to move the `docker-compose-local.yaml` to another repository e.g. `masterportal-scenario-explorer` as it feels more like the point where everything comes together. :thinking:

Reply:
Ok I see! I think it would be best to have two completely seperate compose stacks, one only for UMP with all necessary modules (Geoserver, Keycloak, Postgres, API, Open Policy Agent, ...) and one only for the Masterportal Scenario Explorer Tool. That way backend and frondend are seperated from each other and can be developed seperately in the future as well.

Reply:
This was reverted in favour of: https://github.com/citysciencelab/masterportal-scenario-explorer/pull/2